network-api/networkapi/buyersguide/tests.py

```diff
@@ -333,30 +333,6 @@ def test_localised_homepage(self):
         response = self.client.get('/privacynotincluded/')
         self.assertEqual(response.status_code, 302, 'simple locale gets redirected')
 
-    def test_only_en(self):
```

Comment:
We can't just remove these, we'll have to replace them with new tests that verify the various locales now work as expected.

Reply:
Updated the tests, hopefully that’s covering most cases?